src/client/components/page/page.image.vue

```diff
@@ -22,7 +22,13 @@ export default Vue.extend({
 		};
 	},
 	created() {
-		this.image = this.page.attachedFiles.find(x => x.id === this.value.fileId);
+		if (this.value.hasOwnProperty('fieldId')) {
```

Comment:
I couldn't find an example how to convert an Image to a Media when I call `<x-image>`

Reply:
And actually it's not good. :(